### ISSUE TITLE
Separate pull request checks

### ISSUE THREAD

**Issue by timgraham** [cockroachdb/django-cockroachdb#127]
Currently flake8, isort, and tests are in a single status check. It would be nice to have separate checks, particularly once we add a build for cockroachdb 20.1. Running the tests currently takes about 3 hours so if we have to test with cockroachdb 19.2 and 20.1, the build will take around 6 hours if we can't run them in parallel.

**Comment by rafiss** [comment]
I'm also going to investigate to see if this can be parallelized on the TeamCity side -- I think we can create a "composite" TeamCity job that kicks off other jobs in parallel. That's how the Github CI for cockroachdb/cockroach works.

**Comment by rafiss** [comment]
fixed by https://github.com/cockroachdb/django-cockroachdb/pull/131

To test against 20.1, we should make another script similar to what was done for 19.2 in that PR.
